### ISSUE TITLE
echida/spec-prod does not pick up an svg file when referred to via an <object>

### ISSUE THREAD

**Issue by iherman** [w3c/spec-prod#97]
Consider the [editor's draft of our document](https://w3c.github.io/epub-specs/epub33/overview/), which is within the [epub spec repo](https://github.com/w3c/epub-specs/tree/main/epub33/overview). The spec consists of a respec file, referring to an svg and a png file.

Spec-prod is activated via a [dedicated action](https://github.com/w3c/epub-specs/blob/main/.github/workflows/epub-overview.yml) whose (error) results are [here](https://github.com/w3c/epub-specs/runs/3421453422?check_suite_focus=true). Note that the following lines:

```
$ tar cvf /tmp/echidna.tar *
    Overview.html
    images/
    images/epub.png
```

suggest that spec-prod did pick up the png file and did not pick up the svg file as shown in [line 165](https://github.com/w3c/epub-specs/runs/3421453422?check_suite_focus=true#step:3:165).

I _suspect_ the bug is the following. The [html snippet](https://github.com/w3c/epub-specs/blob/main/epub33/overview/index.html#L98) for referring to the svg file is:

```html
<figure>
	<figcaption>...</figcaption>
	<object data="images/epub.svg" width="600">
		<img src="images/epub.png" width="600" alt="..." />
	</object>
</figure>
```

i.e., the svg file is referred via a `data` attribute rather than an `src`, and the former is not picked up by the process.

---

In the old days, we had an ECHIDNA file in the directory listing all necessary files. The message I got was that the ECHIDNA file should not be used anymore; but maybe that is the situation when we should.  If indeed the ECHIDNA file is required, does it mean that the action yml file should say, at [line 21](https://github.com/w3c/epub-specs/blob/main/.github/workflows/epub-overview.yml#L21)

```
SOURCE: epub33/overview/ECHIDNA
```

cc @sidvishnoi @tidoust @mattgarrish

**Comment by sidvishnoi** [comment]
This would require a little addition to the subresources package (https://github.com/sidvishnoi/subresources) which is used to get all sorts of sub-resources from a page. Svg object image (and probably a few others) is something I missed there, as it's rarely used. I'll try to fix it this week.

Note that Echidna file is not supported by spec-prod at all, nor is there some intention to support it.

**Comment by iherman** [comment]
> Note that Echidna file is not supported by spec-prod at all, nor is there some intention to support it.

Understood.

> Svg object image (and probably a few others) is something I missed there, as it's rarely used. I'll try to fix it this week.

Thanks!

**Comment by sidvishnoi** [comment]
🛠️  Fixed in https://github.com/sidvishnoi/subresources/commit/54284b8deb9fdef64b643ebe9fe999d1f9bc169e and https://github.com/w3c/spec-prod/commit/9bf963ea309eb80abb9e9f9c2734ea63cbbf8fa4.
🚀  Released in v2 ≡ v2.4.2
